### ISSUE TITLE
False positives in `sender.from.phishing.orgbrandprod.map`

### ISSUE THREAD

**Issue by falaix** [martinschaible/rspamd-rules#55]
+ [`/apple.?(™|®|©)?$/i`](https://github.com/martinschaible/rspamd-rules/blob/07eb19c17e4b80e7fca549f3a66e586fbbcbb45b/maps.d/sender/sender.from.phishing.orgbrandprod.map#L67-L67C20) matches valid emails from `Apple <no_reply@email.apple.com>`, e. g. invoices from Apple Store.
+ [`/^meta$/i`](https://github.com/martinschaible/rspamd-rules/blob/07eb19c17e4b80e7fca549f3a66e586fbbcbb45b/maps.d/sender/sender.from.phishing.orgbrandprod.map#L142) matches valid emails from `Meta <security@account.meta.com>`, e. g. login verification like `Did you just log in near $CITY on a new device?`.

**Comment by martinschaible** [comment]
The first one is already fixed.

The second one should be okay. I don't think, that the company uses "meta" as the only part of the senders name.

**Comment by falaix** [comment]
Cool, thanks!

To replicate the Meta case: If you do have a Facebook/Meta account, request a password. Otherwise I'll attach the following snippet for you

```
[...]
X-Priority: 3
X-Mailer: ZuckMail [version 1.00]

From: "Meta" <security@account.meta.com>

Reply-to: noreply <noreply@account.meta.com>
Errors-To: security@account.meta.com
X-Facebook-Notify: password_reset:first_send; mailid=<hidden>
X-FB-Internal-Notiftype: password_reset:first_send
Feedback-ID: password_reset:Facebook
[...]
```

**Comment by martinschaible** [comment]
This bad rule was removed. Thank you for the advice.
